### ISSUE TITLE
How to make SWRevealViewController works like covering the main view not as default pushing the main view ?

### ISSUE THREAD

**Issue by Mohammed-Elias** [John-Lluch/SWRevealViewController#669]
Is there any parameter in the class I can modify to make that happen ?

**Comment by Mohammed-Eliass** [comment]
Is there any parameter in the class I can modify to make that happen ?

**Comment by iDevelopper** [comment]
1) Use right view controller as front and front view controller as menu

2) https://github.com/iDevelopper/PBRevealViewController

**Comment by Mohammed-Elias** [comment]
Thanks but it sounds another library ?

**Comment by iDevelopper** [comment]
Yes, 1. is the solution with SWRevealViewController, 2. is the solution with another library.

**Comment by Mohammed-Elias** [comment]
Good, but what exactly should I do with the storyboard
I have this: 
1) the segue from Reavel View Controller to menu is (sw_rear).
2) the segue from Reavel View Controller to Home is (sw_front).

what should I change ?

**Comment by iDevelopper** [comment]
With SWRevealViewController?

**Comment by Mohammed-Elias** [comment]
yes

**Comment by iDevelopper** [comment]
sw_front from Reveal View Controller to menu.
sw_right from Reveal View Controller to home.

**Comment by Mohammed-Elias** [comment]
forgive me, but I should explain more:
I want Menu VC to cover Home VC and the configuration right now is : 
1) the segue from Reavel View Controller to menu is (sw_rear).
2) the segue from Reavel View Controller to Home is (sw_front).

with these configuration Menu push the Home when I make gesture or tap the button of menu

when I make your changes I got the Menu as the landing view and if I go to Home from the menu I cant go back by the gesture.

**Comment by iDevelopper** [comment]
You have to check some parameters like rightViewRevealWidth, rightViewRevealOverdraw etc...

And add the reveal pan gesture to the reveal controller view.

This is a bit complicated and I recommend to switch to the other library witch is very similar.

**Comment by Mohammed-Eliass** [comment]
Oh, thanks for helping

**Comment by Mohammed-Elias** [comment]
thanks a lot, and by the way your PBRevealViewController is awesome.
